inc/cleantalk-settings.php

```diff
@@ -2679,3 +2684,36 @@ class="apbct-btn-as-link"
                 . __('change email', 'cleantalk-spam-protect') .
             '</button>)';
 }
+
+/**
+ * Implementation of service_update_local_settings functionality
+ */
+add_action('apbct_before_returning_settings', 'apbct__send_local_settings_to_api');
+
+function apbct__send_local_settings_to_api($settings)
+{
+    if (empty($settings) || !is_array($settings)) {
+        return;
+    }
+
+    global $apbct;
+
+    // Settings to JSON
+    $settings = json_encode($settings);
+
+    // Apikey
+    $apikey = $apbct->api_key;
+
+    // Hostname
+    $hostname = preg_replace('/^(https?:)?(\/\/)?(www\.)?/', '', get_site_url());
+
+    Helper::httpRequest(
+        'https://api-next.cleantalk.org',
```

Comment:
why api-next? need to be changed.

Reply:
so it was necessary according to the technical task

Reply:
We can't afford us to use api-next in productive code. Please, revise the task with the boss.